pdns/pdnsutil.cc

```diff
@@ -2734,8 +2748,8 @@ try
     if(bits)
       cerr<<"Requesting specific key size of "<<bits<<" bits"<<endl;
 
-    DNSSECPrivateKey dspk; 
-    shared_ptr<DNSCryptoKeyEngine> dpk(DNSCryptoKeyEngine::make(algorithm)); // defaults to RSA for now, could be smart w/algorithm! XXX FIXME 
+    DNSSECPrivateKey dspk;
+    shared_ptr<DNSCryptoKeyEngine> dpk(DNSCryptoKeyEngine::make(algorithm));
```

Comment:
this seems to only remove the comment

Reply:
Right. [The code already uses P-256](https://github.com/PowerDNS/pdns/pull/7247/files/cd2565d4e93404b0626df05ece8a4dab3a6a4d1f#diff-e69997cee73802012aa0260286fbd5c7R2732) -- the comment was just out-of-date. I'm not sure what the rest of the comment means, but @Habbie said it can go.

Reply:
ah, misunderstood the pr, sorry.